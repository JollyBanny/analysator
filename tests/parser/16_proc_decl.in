procedure MyProc();
var str: string;
begin
  writeln('Some text')
end;

begin
end.