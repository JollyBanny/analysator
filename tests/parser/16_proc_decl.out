program
└──── program block
      ├──── PROCEDURE
      │     ├──── header
      │     │     ├──── myproc
      │     └──── block
      │           ├──── VAR
      │           │     └──── var declaration
      │           │           ├──── str
      │           │           └──── STRING
      │           └──── Compound statement
      │                 └──── Call statement
      │                       └──── writeln
      │                             └──── 'Some text'
      └──── Compound statement
            └──── Empty statement
